build-system/tasks/lint.js

```diff
@@ -132,14 +133,48 @@ function runLinter(path, stream, options) {
  *
  * @return {!Array<string>}
  */
-function filesInPr() {
+function jsFilesInPr() {
   const filesInPr =
         getStdout('git diff --name-only master...HEAD').trim().split('\n');
   return filesInPr.filter(function(file) {
     return path.extname(file) == '.js';
   });
 }
 
+/**
+ * Checks if there are .eslintrc changes in this PR, in which case we must lint
+ * all files.
+ *
+ * @return {boolean}
+ */
+function eslintrcChangesInPr() {
```

Comment:
Consider checking that this is a PR build inside this function instead.

Reply:
The check for a PR build needs to be outside because it's also used when `.eslintrc` is _not_ in the list of files.

Reply:
On second thought, good idea. Done.